index.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
 
     let parentOptions = this._getParentOptions();
 
-    this.addonOptions = parentOptions.emberArgumentDecorators || {};
+    this.addonOptions = parentOptions.emberDecoratorsArgument || {};
```

Comment:
For index.js/ember-cli configuration the convention is generally the package name these days, so I think:

```js
this.addonOptions = parentOptions['@ember-decorators/argument'] || {};
```

I can't find any super solid conventions for runtime configuration, especially for projects like `@ember-decorators` that will consist of many smaller packages. For now, I think it would be fine to match them to the package name as well to keep things consistent.